lcu_driver/connection.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, connector, process_or_string: Union[Process, str]):
         self._connector = connector
         self._ws = None
         self.locals = {}
+        self.closed = False
```

Comment:
`closed` should be defined on `init` method to reset it's value every time it gets called